gpkit/nomials/data.py

```diff
@@ -54,13 +50,6 @@ def __hash__(self):
                                          [str(self.units)]))
         return self._hashvalue
 
-    @classmethod
-    def fromnomials(cls, nomials):
-        """Construct a NomialData from an iterable of Signomial objects"""
-        nd = cls()  # use pass-through version of __init__
-        nd.init_from_nomials(nomials)
```

Comment:
should we also drop the `init_from_nomials` method? I think this is the only place it was used